src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudSnapshotCreatedIT.java

```diff
@@ -78,7 +78,7 @@ public static void init() throws Exception {
     ComputeEngineCloud cloud = initCloud(jenkinsRule);
     client = initClient(jenkinsRule, label, log);
 
-    assertTrue(cloud.configurations.isEmpty());
+    assertTrue(cloud.getConfigurations().isEmpty());
```

Comment:
With the constructor used here `cloud.getConfigurations()` is null, which triggers an NPE on this line. Recommend `assertNull(cloud.getConfigurations())` or removing this line entirely.